### ISSUE TITLE
The NEW CP1 can not be rooted

### ISSUE THREAD

**Issue by jerjupiter** [HappyZ/dpt-tools#141]
Hi HappyZ, I have a CP1(Chinese version, Version: 1.6.02.06240) made after September.
And I try the step 0x2 of the rooting guide, but there is nothing printed, and the CP1 just restart and claim the update failed, and the diagnosis mode did not show a black square. 
Does Sony fix the leak and CP1 can not be rooted now?

**Comment by HappyZ** [comment]
That's a good question. I haven't been following up with newer devices, but
it should relate with the system firmware. If nothing is printed and
directly jumps back at rebooting, it is likely they've patched it.

But on the other hand since there is no new system update, I wonder about
their actions on older devices. In theory they can send an OTA update
easily. It doesn't make sense for them to only fix new devices without
taking care of the older ones.

**Comment by jerjupiter** [comment]
I exchange a new one, and it still can not be rooted. Maybe Sony really has fix the leak.
Somebody tell me the machine may be modified before...but I think the probability is low..

**Comment by sekkit** [comment]
I must say, I ordered a new device from jd.com and it worked well.

**Comment by HappyZ** [comment]
@sekkit  `jd.com` may still have earlier produced devices. Not sure if they've fixed with the new batch of produced devices. It would be really funny if they did not push a software fix but rather a "hardware" fix. If the "hardware" fix is true, then I think a software fix may come soon.

**Comment by 78dk** [comment]
9月后面生产的在官方网买的国行RP1也是无法破解的

**Comment by duoduodad** [comment]
Hi , I also have a RP1(Chinese version, Version: 1.6.03.09261) made after September.
And I try the step 0x2 of the rooting guide, but there is nothing printed, and the RP1 just restart and claim the update failed, and the diagnosis mode did not show a black square.
Does Sony fix the leak and RP1 can not be rooted now?

**Comment by HappyZ** [comment]
Yes. And for these new devices after September, yes.

**Comment by yefeiyu** [comment]
How to check the factory date？

**Comment by xpierrohk** [comment]
I bought a DPT-RP1 last month:
- Made in China
- Bought in the US via grey importer
- Delivered to Hong Kong (so the baby came home in a way...)

I was able to root it. 

How can we check manufacture date ? How do we know it's "after september" ?

**Comment by rushsaker** [comment]
> I bought a DPT-RP1 last month:
> 
> * Made in China
> * Bought in the US via grey importer
> * Delivered to Hong Kong (so the baby came home in a way...)
> 
> I was able to root it.
> 
> How can we check manufacture date ? How do we know it's "after september" ?

the manufacture date is printed on the back of the case.

**Comment by rushsaker** [comment]
I bought a DPT-RP1 today manufactured in january 2020. unable to root T.T

**Comment by xpierrohk** [comment]
> I bought a DPT-RP1 today manufactured in january 2020. unable to root T.T

What happens? No text display whatsoever when running the root update package ? Or you see the root doing stuff but no diag mode at the end ?

I suppose you wouldn't consider sacrificing yours to dump the eMMC chip? :D Like HappyZ I still can't believe they patched the update script at the factory but not in the latest updates, so I wonder if they just didnt harden the diag mode...

**Comment by rushsaker** [comment]
Yes, there is nothing on the screen when running the root update package  just restart and claim the update failed. Then it's not rooted after the update.

**Comment by wangfreon** [comment]
Is there anyone who knows there are still some active discussions on the rooting of devices produced after Sep. 2019?

**Comment by LichunZhang** [comment]
I ask some sellers on Taobao and they confirm that the devices after Sep. 2019 can not be rooted by using the approach here. But they still have methods to reboot it locally which means we need to send the device to them to operate. Thank you.

**Comment by grayguest** [comment]
> I ask some sellers on Taobao and they confirm that the devices after Sep. 2019 can not be rooted by using the approach here. But they still have methods to reboot it locally which means we need to send the device to them to operate. Thank you.

but the price is so high.

**Comment by Rocknokokoro** [comment]
Hi, I have a DPT-RP1 from late 2017 or early 2018, I don't remember exactly, but definitely pre 2019. The device is updated to firmware version 1.6.50.14130. 

When I run dpt-tools fw, nothing is printed on screen, my device just restarts and shows the update failed message. From what I read in these threads, the update should have not interfered with the patch, is this correct? If this case, either I am doing something wrong or some pre 2019 devices are also unrootable. Any tips on either direction? Thanks!

**Comment by zhangjunda23** [comment]
is there any solution for the devices made after Sep. 2019?

**Comment by M1thrandirZ** [comment]
Is there any solution for the devices made after Sep. 2019? I have a dpt-rp1 made on Oct. 2020, Chinese version. Some sellers on Taobao tell me that the device can only be rooted by sending to them.

**Comment by Sfinx** [comment]
> Hi, I have a DPT-RP1 from late 2017 or early 2018, I don't remember exactly, but definitely pre 2019. The device is updated to firmware version 1.6.50.14130.
> 
> When I run dpt-tools fw, nothing is printed on screen, my device just restarts and shows the update failed message. From what I read in these threads, the update should have not interfered with the patch, is this correct? If this case, either I am doing something wrong or some pre 2019 devices are also unrootable. Any tips on either direction? Thanks!

The same here - device is from 2017, came with ancient 1.4.01.16100. No luck with rooting - just reboots with famous failing message. Upgraded it to official 1.6.50.14130 - the same sh.t.

**Comment by chengcic** [comment]
Any update on this issue. Seems there is little information regarding root failure on the latest model. :(
